Update how upgrades can be triggered by user (#3933)
timescale/docs: use-timescale/upgrades.md

## Non-critical maintenance updates

Non-critical upgrades are available before the upgrade is performed
automatically by $CLOUD_LONG. To upgrade $TIMESCALE_DB manually, run `ALTER
EXTENSION timescaledb UPDATE` in your $SERVICE_LONG, or `Pause` and `Resume`.

> This triggers the upgrade in the next available maintenance window. You can

Author: ```suggestion
If no action is taken by the user, the upgrade is triggered in the next available maintenance window. You can
```

Author: @billy-the-fish with all the suggested changes committed, the meaning I wanted to convey got distorted a bit. Could you have a look at this sentence?

## Non-critical maintenance updates

> Non-critical upgrades are made available before the upgrade is performed

Reviewer: ```suggestion
Non-critical upgrades are available before the upgrade is performed
```

> automatically. During this time you can run `ALTER EXTENSION timescaledb

Reviewer: ```suggestion
automatically by $CLOUD_LONG. To upgrade $TIMESCALE_DB manually, run `ALTER EXTENSION timescaledb
```

> UPDATE` in your in your database or `Pause` and `Resume` your service. The

Reviewer: ```suggestion
UPDATE` in your in your $SERVICE_LONG, or `Pause` and `Resume` your service.
```

> upgrade is triggered automatically in the next available maintenance window for

Reviewer: ```suggestion
This triggers the upgrade in the next available maintenance window.
```

> your service if you haven't taken any action to do the upgrade yourself. You

Reviewer: Remove this line.

> can configure the maintenance window so that these upgrades are started only at

Reviewer: ```suggestion
You can configure the maintenance window so that these upgrades are started at
```

> a particular time, on a set day of the week. If there are no pending upgrades

Reviewer: ```suggestion
a particular time, on a set day of the week. 

If there are no pending upgrades
```
